docs/changelog-fragments.d/164.doc.md

```diff
@@ -1,5 +1,5 @@
-Dropped the brackets from the changelog titles for the release sections.
-We now don't strictly follow the release notes format suggested by [Keep
-a Changelog] -- by {user}`webknjaz`
+Dropped the brackets from the changelog titles for the release sections. We now
+don't strictly follow the release notes format suggested by [Keep a Changelog]
+-- by {user}`webknjaz`
 
-[Keep a Changelog]: https://keepachangelog.com/en/1.1.0/
+[keep a changelog]: https://keepachangelog.com/en/1.1.0/
```

Comment:
This is particularly weird — it's hard to search for non-matching names because of the differing case.